### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by auras** [arturbosch/sonar-kotlin#24]
```
[12:57:14]	[Step 2/2] ERROR: Error during Sonar runner execution
[12:57:14]	[Step 2/2] org.sonar.runner.impl.RunnerException: Unable to execute Sonar
[12:57:14]	[Step 2/2] at org.sonar.runner.impl.BatchLauncher$1.delegateExecution(BatchLauncher.java:91)
[12:57:14]	[Step 2/2] at org.sonar.runner.impl.BatchLauncher$1.run(BatchLauncher.java:75)
[12:57:14]	[Step 2/2] at java.security.AccessController.doPrivileged(Native Method)
[12:57:14]	[Step 2/2] at org.sonar.runner.impl.BatchLauncher.doExecute(BatchLauncher.java:69)
[12:57:14]	[Step 2/2] at org.sonar.runner.impl.BatchLauncher.execute(BatchLauncher.java:50)
[12:57:14]	[Step 2/2] at org.sonar.runner.api.EmbeddedRunner.doExecute(EmbeddedRunner.java:102)
[12:57:14]	[Step 2/2] at org.sonar.runner.api.Runner.execute(Runner.java:100)
[12:57:14]	[Step 2/2] at org.sonar.runner.Main.executeTask(Main.java:70)
[12:57:14]	[Step 2/2] at org.sonar.runner.Main.execute(Main.java:59)
[12:57:14]	[Step 2/2] at org.sonar.runner.Main.main(Main.java:53)
[12:57:14]	[Step 2/2] Caused by: java.lang.IllegalArgumentException: -1 is not a valid line for a file
[12:57:14]	[Step 2/2] at org.sonar.api.internal.google.common.base.Preconditions.checkArgument(Preconditions.java:145)
[12:57:14]	[Step 2/2] at org.sonar.api.batch.fs.internal.DefaultInputFile.checkValid(DefaultInputFile.java:288)
[12:57:14]	[Step 2/2] at org.sonar.api.batch.fs.internal.DefaultInputFile.newPointer(DefaultInputFile.java:226)
[12:57:14]	[Step 2/2] at org.sonar.api.batch.fs.internal.DefaultInputFile.selectLine(DefaultInputFile.java:249)
[12:57:14]	[Step 2/2] at io.gitlab.arturbosch.detekt.sonar.sensor.DetektSensor.primaryLocation(DetektSensor.kt:78)
[12:57:14]	[Step 2/2] at io.gitlab.arturbosch.detekt.sonar.sensor.DetektSensor.reportIssue(DetektSensor.kt:63)
[12:57:14]	[Step 2/2] at io.gitlab.arturbosch.detekt.sonar.sensor.DetektSensor.access$reportIssue(DetektSensor.kt:22)
[12:57:14]	[Step 2/2] at io.gitlab.arturbosch.detekt.sonar.sensor.DetektSensor$reportIssues$1.accept(DetektSensor.kt:51)
[12:57:14]	[Step 2/2] at io.gitlab.arturbosch.detekt.sonar.sensor.DetektSensor$reportIssues$1.accept(DetektSensor.kt:22)
[12:57:14]	[Step 2/2] at java.util.TreeMap.forEach(TreeMap.java:1005)
[12:57:14]	[Step 2/2] at io.gitlab.arturbosch.detekt.sonar.sensor.DetektSensor.reportIssues(DetektSensor.kt:49)
[12:57:14]	[Step 2/2] at io.gitlab.arturbosch.detekt.sonar.sensor.DetektSensor.execute(DetektSensor.kt:33)
[12:57:14]	[Step 2/2] at org.sonar.scanner.sensor.SensorWrapper.analyse(SensorWrapper.java:53)
[12:57:14]	[Step 2/2] at org.sonar.scanner.phases.SensorsExecutor.executeSensor(SensorsExecutor.java:87)
[12:57:14]	[Step 2/2] at org.sonar.scanner.phases.SensorsExecutor.execute(SensorsExecutor.java:81)
[12:57:14]	[Step 2/2] at org.sonar.scanner.phases.SensorsExecutor.execute(SensorsExecutor.java:67)
[12:57:14]	[Step 2/2] at org.sonar.scanner.phases.AbstractPhaseExecutor.execute(AbstractPhaseExecutor.java:75)
[12:57:14]	[Step 2/2] at org.sonar.scanner.scan.ModuleScanContainer.doAfterStart(ModuleScanContainer.java:178)
[12:57:14]	[Step 2/2] at org.sonar.core.platform.ComponentContainer.startComponents(ComponentContainer.java:144)
[12:57:14]	[Step 2/2] at org.sonar.core.platform.ComponentContainer.execute(ComponentContainer.java:129)
[12:57:14]	[Step 2/2] at org.sonar.scanner.scan.ProjectScanContainer.scan(ProjectScanContainer.java:259)
[12:57:14]	[Step 2/2] at org.sonar.scanner.scan.ProjectScanContainer.scanRecursively(ProjectScanContainer.java:254)
[12:57:14]	[Step 2/2] at org.sonar.scanner.scan.ProjectScanContainer.scanRecursively(ProjectScanContainer.java:252)
[12:57:14]	[Step 2/2] at org.sonar.scanner.scan.ProjectScanContainer.doAfterStart(ProjectScanContainer.java:243)
[12:57:14]	[Step 2/2] at org.sonar.core.platform.ComponentContainer.startComponents(ComponentContainer.java:144)
[12:57:14]	[Step 2/2] at org.sonar.core.platform.ComponentContainer.execute(ComponentContainer.java:129)
[12:57:14]	[Step 2/2] at org.sonar.scanner.task.ScanTask.execute(ScanTask.java:47)
[12:57:14]	[Step 2/2] at org.sonar.scanner.task.TaskContainer.doAfterStart(TaskContainer.java:86)
[12:57:14]	[Step 2/2] at org.sonar.core.platform.ComponentContainer.startComponents(ComponentContainer.java:144)
[12:57:14]	[Step 2/2] at org.sonar.core.platform.ComponentContainer.execute(ComponentContainer.java:129)
[12:57:14]	[Step 2/2] at org.sonar.scanner.bootstrap.GlobalContainer.executeTask(GlobalContainer.java:118)
[12:57:14]	[Step 2/2] at org.sonar.batch.bootstrapper.Batch.executeTask(Batch.java:117)
[12:57:14]	[Step 2/2] at org.sonar.batch.bootstrapper.Batch.execute(Batch.java:77)
[12:57:14]	[Step 2/2] at org.sonar.runner.batch.IsolatedLauncher.execute(IsolatedLauncher.java:48)
[12:57:14]	[Step 2/2] at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[12:57:14]	[Step 2/2] at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[12:57:14]	[Step 2/2] at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[12:57:14]	[Step 2/2] at java.lang.reflect.Method.invoke(Method.java:498)
[12:57:14]	[Step 2/2] at org.sonar.runner.impl.BatchLauncher$1.delegateExecution(BatchLauncher.java:87)
[12:57:14]	[Step 2/2] ... 9 more
[12:57:14]	[Step 2/2] ERROR:
[12:57:14]	[Step 2/2] ERROR: Re-run SonarQube Runner using the -X switch to enable full debug logging.
[12:57:14]	[Step 2/2] Process exited with code 1
[12:57:14]	[Step 2/2] Process exited with code 1
[12:57:14]	[Step 2/2] Step Static Analysis (SonarQube Runner) failed
```

**Comment by auras** [comment]
This was with 0.2.2. Was this fixed already in 0.2.3?

**Comment by auras** [comment]
Nevermind, seems to be fixed in 0.2.3
